browser/importer/brave_external_process_importer_client.cc

```diff
@@ -14,7 +14,8 @@
 
 namespace {
 bool ShouldUseBraveImporter(importer::ImporterType type) {
-  if (type == importer::TYPE_CHROME || type == importer::TYPE_EDGE_CHROMIUM)
+  if (type == importer::TYPE_CHROME || type == importer::TYPE_EDGE_CHROMIUM ||
```

Comment:
If we'll be adding more soon (ex: Opera), might be cleaner as a `switch` statement
```cpp
  switch (type) {
    case importer::TYPE_CHROME:
    case importer::TYPE_EDGE_CHROMIUM:
    case importer::TYPE_VIVALDI:
      return true;
  }
  return false;
```

Reply:
done